src/actions/heap/heap.ts

```diff
@@ -386,8 +461,8 @@ export class HeapAction extends Hub.Action {
         })
         .promise()
     } catch (err) {
-      errors.push(err)
-    }
+      errors.push(new Error("Encountered an error in sending request to heap, error: " + getErrorMessage(err)))
```

Comment:
Can we make this a debug log?

Reply:
It will be logged in the end, when the whole thing is completed. The original test verifies the error. I want to make it an InternalError, so that the error is not included in the final response.